cpu/nrf52/periph/spi.c

```diff
@@ -43,12 +43,14 @@ static mutex_t locks[SPI_NUMOF];
  * @brief   array with a busy mutex for each SPI device, used to block the
  *          thread until the transfer is done
  */
-static mutex_t busy[SPI_NUMOF];
+// static mutex_t busy[SPI_NUMOF];
```

Comment:
Why are you removing mutex from the nrf52 SPI driver?
This looks pretty hacky. (not to mention that the radio module can be hooked up to other CPUs as well)

Reply:
It is hacky there is a FIXME commit there.

Reply:
150a798

Reply:
I definitively need to fix this.

Reply:
Reverted will remove from history once rebasing.